packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -349,6 +349,10 @@ export function TransferPanel() {
       return
     }
 
+    if (!isEOA && !isSmartContractWallet) {
+      throw 'Account type is undefined'
```

Comment:
Maybe we should just `return` instead of throwing?

Reply:
sure, updated